hathor/cli/generate_genesis.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Hathor Labs
```

Comment:
Outdated year.

Reply:
Fixed in https://github.com/HathorNetwork/hathor-core/pull/1061/commits/a750cbd94afddebb8e1d42494c61196e8b1cf5e4